inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,35 @@
 """Remove the user's remote storage contents via CLI."""
 
+import sys
+
 from inductiva import storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print("inductiva storage remove: error: "
+              "argument path not allowed with argument --all")
+        sys.exit(1)
 
     if not confirm:
-        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        if all_paths:
+            prompt = input("Are you sure you want to remove everything "
+                           "from your storage(y/[N])")
+        else:
+            prompt = input(f"Are you sure you want to remove {len(paths)}"
```

Comment:
Since our id's have no meaning and are just random numbers and letters I don't think that showing a list of ids like this: n1tuy5pe5wo32ihzz8zar58cn,tgnasiw09a8jprbwwkq2jxiry,ir7rex4rwo4nktv9qjikr0uf3,se4ozwx5uqpq5i3gymp1jkcoa 
would be very helpful. 

Furthermore, we are using the same method for the tasks kill
Do you confirm you want to kill {len(args.id)} tasks (y/[N])?

I can change it on both, but I think a list of id's won't help the users.

Reply:
That is because we have not yet started organising our simulation into named folders. But when running a simulation you can pass a `storage_dir="happily_ever_after"` and the simulations will be saved in that folder in the bucket. 

I haven't seen the `kill` method. But I really think we could do it on both. Not in a single line though. Something like:
```bash
"Path to be removed: 
> n1tuy5pe5wo32ihzz8zar58cn
> tgnasiw09a8jprbwwkq2jxiry
> ir7rex4rwo4nktv9qjikr0uf3
> se4ozwx5uqpq5i3gymp1jkcoa
Are you sure you want to remove the above  paths from your storage? (y/[N])"
```

Reply:
Changing this here would mean we have to change kill too (kill was just merged). @sergiomsantos do I have the green light to change it?

Reply:
Changed it